addons/addon-base-post-deployment/packages/base-post-deployment/lib/steps/create-root-user-service.js

```diff
@@ -60,9 +60,8 @@ class CreateRootUserService extends Service {
       });
       this.log.info('Created root user in the data lake');
 
-      await dbPasswordService.savePassword(getSystemRequestContext(), {
+      await dbPasswordService.saveRootPassword(getSystemRequestContext(), {
         uid: createdUser.uid,
-        username: rootUserName,
```

Comment:
It seems odd that we need to pass `root` username when the method itself is called `saveRootPassword`

Reply:
We're not passing the `root` username right? The `root` username is being deleted. We're just passing in the `rootUserPassword`

Reply:
oops sorry, makes sense :)